applications/crossbar/priv/couchdb/schemas/devices.json

```diff
@@ -21,6 +21,20 @@
                     "description": "Enable the call-forwarding parameters if the device is offline",
                     "type": "boolean"
                 },
+                "hotdesk": {
```

Comment:
I don't think this schema should be under "call_forward" but on the root of the doc, no?